scripts/examples/ota_test.sh

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env bash
+
```

Comment:
shoud this script `set -e` to exist whenever a command fails? Unsure if the `pkill` commands will work in that case though. 

But it does sound like we want to exit the script if something like build or ota image creation fails.

Reply:
Hmm. This is a good point. I will look into a way to exit the script, while making sure processes are kill if they've been started already.